gen/ip-detect/vagrant6.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Can we test this out with `dcos-vagrant`?

Reply:
yes

Reply:
Thanks !!